src/viewport/index.tsx

```diff
@@ -207,28 +208,31 @@ class Viewport extends React.Component<ViewportProps, ViewportState> {
         simulariumController.zoomOut = this.zoomOut;
         simulariumController.visGeometry = this.visGeometry;
         simulariumController.trajFileInfoCallback = (
-            msg: TrajectoryFileInfo
+            msg: TrajectoryFileInfoAny
         ) => {
-            this.visGeometry.handleTrajectoryData(msg);
-            // handleTrajectoryData() above creates a new bounding box and tick marks
-            // (via resetBounds()) and sets VisGeometry.tickIntervalLength, which is now
-            // available for use as the length of the scale bar in the UI.
+            // Update TrajectoryFileInfo format to latest version
+            const newMsg: TrajectoryFileInfo = updateTrajectoryFileInfoFormat(msg)
```

Comment:
😢

Reply:
the sadness is only because this update call can't be done at a lower level when the data message is first received